styx-scheduler-service/src/main/java/com/spotify/styx/TriggerManager.java

```diff
@@ -152,7 +152,8 @@ private void tryTriggering(Workflow workflow,
   private void tryTriggering0(Workflow workflow,
                                  TriggerInstantSpec instantSpec,
                                  Set<WorkflowId> enabledWorkflows) {
-    if (enabledWorkflows.contains(workflow.id())) {
+    // Ignore flyte workflows
+    if (enabledWorkflows.contains(workflow.id()) && workflow.configuration().flyteExecConf().isEmpty()) {
```

Comment:
I think it makes sense to remove the flyteExecConf from this if check, and instead do an early return with it's own debug log